test-framework/junit5-internal/src/main/java/io/quarkus/test/QuarkusUnitTest.java

```diff
@@ -424,7 +424,9 @@ public void execute(BuildContext context) {
                     }
                 });
             } catch (ClassNotFoundException e) {
-                //ignore
+                System.err.println("Couldn't make the test class " + testClass.getSimpleName() + " an unremovable bean"
```

Comment:
Any chance it could use the `rootLogger` instead of `System.err`?

Reply:
Unfortunately no. Everytime I've tried that, it never worked.